src/main/scala/nl.knaw.dans.easy.deposit/package.scala

```diff
@@ -15,17 +15,20 @@
  */
 package nl.knaw.dans.easy
 
-import java.nio.file.{ FileAlreadyExistsException, Paths }
+import java.io.IOException
+import java.nio.file.spi.FileSystemProvider
+import java.nio.file.{ FileAlreadyExistsException, Paths, StandardCopyOption }
 import java.util.UUID
 
-import better.files.StringOps
+import better.files.{ File, StringOps }
 import nl.knaw.dans.bag.DansBag
 import nl.knaw.dans.easy.deposit.docs.StateInfo.State.State
+import nl.knaw.dans.lib.logging.DebugEnhancedLogging
 
 import scala.util.{ Failure, Try }
 import scala.xml._
 
-package object deposit {
+package object deposit extends DebugEnhancedLogging {
```

Comment:
no debug logging necessary anymore, as well as unused imports

```suggestion
package object deposit {
```